### ISSUE TITLE
allow multiple games to be happening at once

### ISSUE THREAD

**Issue by andyshinn** [mrooney/dobbins-or-bobbins#10]
This will require separating game and bot logic a little bit more and associating a game with a certain guild or guild and channel.

**Comment by mrooney** [comment]
Yes I thought about this! Actually I think all it needs is a dictionary of Game objects where the keys are the channels. And if PMs are still supported, you'd probably want another dict that is player => game that you add to when they say "in!"
